### PR TITLE
Copy paste

### LINKED ISSUE #45: Implement copy & paste on the canvas

### PR DESCRIPTION
**Pull request by jeffswenson**
Resolves #45

### CODE REVIEW

**Comment by tlein** [comment]
OSX shows a border around the canvas if you make it contentEditable. You should add this styling:

`.canvas-container canvas, .canvas-container canvas:focus {
    outline: 0 solid transparent;
}
`

Jeff: Updated

File: src/duckling/main.ts [hunk 1]
```diff
@@ -91,13 +83,14 @@ bootstrap(ShellComponent, [
     provideInstance(attributeDefaultService, AttributeDefaultService),
     provideInstance(entityPositionSetService, EntityPositionSetService),
     provideInstance(entitySystemService, EntitySystemService),
-    provideInstance(entityCreatorTool, EntityCreatorTool),
```
Comment (tlein) [inline src/duckling/main.ts:94]: Was having this here originally a mistake?

**Comment by tlein** [comment]
clipboard-events.service.ts is empty, should it be deleted?

Jeff: Deleted

File: src/duckling/main.ts (hunk 1 above)
Comment (jeffswenson) [inline src/duckling/main.ts:94]: Yes

**Pull request merged**